src/pyodide-py/pyodide/webloop.py

```diff
@@ -65,7 +65,7 @@ def run_forever(self):
         """
         pass
 
-    def run_until_complete(self, future: Awaitable):
+    def run_until_complete(self, future: Awaitable):  # type: ignore
```

Comment:
mypy errors saying that the signature doesn't match the Abstract class, but I don't see any issues there https://github.com/python/cpython/blob/920bf6a3a656e329c2bcbb761eb8c13c46c8cd05/Lib/asyncio/events.py#L214